packages/core/types/Data.tsx

```diff
@@ -52,3 +52,5 @@ export type Data<
   content: Content<Props>;
   zones?: Record<string, Content<Props>>;
 };
+
+export type MetaData = { [key: string]: any };
```

Comment:
```suggestion
export type Metadata = { [key: string]: any };
```

I also think you need to run `npm run format` and commit the results to fix various formatting issue.